mavis/DTable.h

```diff
@@ -240,13 +240,14 @@ class DTable
      * @return
      */
     template<class InstTypeAllocator, typename ...ArgTypes>
-    typename InstType::PtrType makeInstDirectly(const ExtractorDirectInfoIF &ex_info,
+    typename InstType::PtrType makeInstDirectly(const ExtractorDirectInfoIF &ex_info,const std :: map<mavis :: OpcodeInfo::SpecialField,uint64_t>& special_fields_map_,
```

Comment:
Bizarre spacing on the `::` above.  As Kathlene mentioned, please follow existing formatting.

Reply:
This breaks backward compatibility with existing users of this function.

Reply:
Actually this looks like it might be leftover from the other PR. Since you added special fields to `ExtractorDirectInfo` you don't need to modify this method signature at all - right?